<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addeui</class>
 <widget class="QDialog" name="addeui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加EUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>1.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>11.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit11"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>2.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit2"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>12.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit12"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>3.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit3"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>13.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit13"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>4.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit4"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>14.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit14"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>5.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit5"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>15.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit15"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>6.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit6"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>16.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEdit16"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>7.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit7"/>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>17.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="lineEdit17"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>8.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit8"/>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>18.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="lineEdit18"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>9.</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit9"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>19.</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLineEdit" name="lineEdit19"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>10.</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit10"/>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>20.</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLineEdit" name="lineEdit20"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Send_funtion">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
